appendix-basic65.tex

```diff
@@ -7355,13 +7351,13 @@ \subsection{RPEN}
 \item [Format:] {\bf RPEN(n)}
 \item [Usage:]  Returns the colour index of pen n.
 
-                {\bf n} PEN number (0-2).
+                {\bf n} pen number (0-2).
 
-{\ttfamily
-                0: draw pen \\
-                1: erase pen \\
-                2: outline pen
-}
+                \begin{itemize}
+                    \item {0} draw pen.
+                    \item {0} erase pen.
+                    \item {0} outline pen.
```

Comment:
In the transition to \item, these all became 0 (should be 0, 1, 2).

Hmm, it's interesting that pen 1 is described as the 'erase pen' here, as back in the `PEN` command's description, it was described as `off bits in jam2 mode`.

I'm not too familiar with jam2 mode as yet, and I don't think we've got any good docs for it yet, so it might require some digging into the ROM-source and experimentation with the commands to get a feel for this. Will @ mention @Edilbert in-case he has any insights.

Reply:
too much copy/paste from me